### PR TITLE
Satya fix button spacing

### PR DESCRIPTION
**Pull request by SatyashanthiT**
Description:
This PR addresses a UI bug in the WBSTasks component where buttons were stacking vertically without appropriate spacing on mobile and tablet views. The fix ensures that the buttons are now properly spaced in mobile view, improving overall appearance and usability. The layout works consistently in both light and dark mode.

Related PRs (if any):
This frontend PR is related to the PR2679 frontend PR.

Main Changes Explained:
Updated the WBSTasks.jsx component to organize buttons inside a container for better alignment.
Added CSS rules in wbs.css to apply responsive margin between buttons when viewed on mobile and tablet screens.
Ensured the fix applies to both light and dark mode, maintaining the visual experience.

How to Test:
Check into the current branch.
Run the following commands:
npm install
npm start
Clear your site data/cache.
Log in as an admin user.
Navigate to Dashboard →Other Links→  Projects→ Select WBS.

Verify that:
The buttons now have proper spacing when viewed on mobile and tablet devices.
The button layout works as expected in both light mode and dark mode.
Ensure that the buttons are spaced properly and are not overlapping or stacking incorrectly in smaller screens.

Screenshots or Videos of Changes:

https://github.com/user-attachments/assets/b87d82b8-111f-496f-a005-e258835774ce

### CODE REVIEW

**Review by shivansh-nathani — APPROVED** [review]
I have tested the code and things are working as expected, checked the same for different device sizes and buttons are equally spaced and aren't overlapping.

https://github.com/user-attachments/assets/ccbd8965-262c-4e47-921e-cb9e776d0a85

**Review by nikhilpittala16 — DISMISSED** [review]
I have tested the PR and everything works as expected.

https://github.com/user-attachments/assets/fad0adca-d14f-40d8-b9d4-b2455c41512b

**Review by NK3101 — APPROVED** [review]
Tested the PR, the buttons are correctly spaced, with the layout functioning as expected in both light and dark modes. The buttons do not overlap or stack incorrectly on smaller screens.

https://github.com/user-attachments/assets/5dc6538c-c886-4a1a-acfd-d8be79f519d8

**Comment by ShreyaLaheri** [comment]
I tested the changes, works well in both light and dark mode
image
image

**Review by CodewithKoushica — APPROVED** [review]
The fix for the button stacking issue in the WBSTasks component works well overall. I tested it on both mobile and tablet views, and the button alignment and spacing now look much cleaner and more user-friendly. The responsiveness across different screen sizes is improved, and the layout holds up nicely in both light and dark modes.

Screenshot 2024-10-22 at 11 01 11 AM

File: src/components/Projects/WBS/WBSDetail/wbs.css [hunk 1]
```diff
@@ -234,10 +234,15 @@
   padding-left: .25em;
 }
 .tasks-detail-task-name { /* task column height and width changes*/
-  
+  width: 35%; 
+.tasks-detail-task-name { /* task column height and width changes*/
   height: auto; 
 }
```
Comment (Muhideenthegreat) [inline src/components/Projects/WBS/WBSDetail/wbs.css:240]: Missing curly brackets here

**Review by Muhideenthegreat — APPROVED** [review]
Verification: 

Outside of the missing curly brackets, the various buttons are now in a container and work as they should. They are also responsive to various screen sizes.

Screenshot 2024-10-25 at 6 11 31 PM
Screenshot 2024-10-25 at 6 12 05 PM

**Review by nikhilgiri0226 — APPROVED** [review]
I have tested the PR for different size of screens and the buttons aren't overlapping. Tested the same for different size of screens. Kindly find the screenshots below. Also, there was a missing "}" curly brace in wbs.css file. 
image

image

image

image

**Pull request merged**